cmd/gocritic/testdata/unusedParams/negative_tests.go

```diff
@@ -0,0 +1,12 @@
+package checker_test
+
+func (f *foo) g1(x int, _ float64) {
```

Comment:
I think it's intentional that receiver parameter not checked.
Makes sense.

But we should document that explicitly, because it's possible to do:
```go
func (*foo) g1(x int, _ float64) {}
```

Reply:
Good point